packages/dnb-eufemia/src/components/date-picker/__tests__/DatePicker.test.tsx

```diff
@@ -223,6 +223,264 @@ describe('DatePicker component', () => {
     ).not.toContain('dnb-date-picker--opened')
   })
 
+  it('should delete input content one number at a time when `date` is prop controlled', async () => {
+    const Component = () => {
+      const [date, setDate] = useState('2024-05-17')
+
+      return (
+        <DatePicker
+          showInput
+          date={date}
+          onChange={({ date }) => setDate(date)}
+        />
+      )
+    }
+
+    render(<Component />)
+
+    const [day, month, year]: Array<HTMLInputElement> = Array.from(
+      document.querySelectorAll('input.dnb-input__input')
+    )
+
+    expect(day.value).toBe('17')
+    expect(month.value).toBe('05')
+    expect(year.value).toBe('2024')
+
+    await userEvent.click(year)
+    await userEvent.keyboard('{ArrowRight>4}{backspace}')
```

Comment:
```suggestion
    await userEvent.keyboard('{ArrowRight>4}{Backspace}')
```
For consistency I would write every command in PascalCase.

Reply:
Improved in commit https://github.com/dnbexperience/eufemia/pull/4342/commits/ef4680a204c700f1a83a33e897fbcd278d7509fc